install.sh

```diff
@@ -313,27 +313,34 @@ install_msgpack_from_source( )
 
 install_blis()
 {
-    #Download prebuilt AMD multithreaded blis
-    if [[ ! -e "${build_dir}/deps/blis/lib/libblis.a" ]]; then
-      case "${ID}" in
-          centos|rhel|sles|opensuse-leap|almalinux)
-              wget -nv -O blis.tar.gz https://github.com/amd/blis/releases/download/2.0/aocl-blis-mt-centos-2.0.tar.gz
-              ;;
-          ubuntu)
-              wget -nv -O blis.tar.gz https://github.com/amd/blis/releases/download/2.0/aocl-blis-mt-ubuntu-2.0.tar.gz
-              ;;
-          *)
-              echo "Unsupported OS for this script"
-              wget -nv -O blis.tar.gz https://github.com/amd/blis/releases/download/2.0/aocl-blis-mt-ubuntu-2.0.tar.gz
-              ;;
-      esac
-
-      tar -xvf blis.tar.gz
-      rm -rf blis/amd-blis-mt
-      mv amd-blis-mt blis
-      rm blis.tar.gz
-      cd blis/lib
-      ln -sf libblis-mt.a libblis.a
+    if [[ ! -e "/opt/AMD/aocl/aocl-linux-gcc-4.2.0/gcc/lib_ILP64/libblis-mt.a" ]] &&
+        [[ ! -e "/opt/AMD/aocl/aocl-linux-aocc-4.1.0/aocc/lib_ILP64/libblis-mt.a" ]] &&
+        [[ ! -e "/opt/AMD/aocl/aocl-linux-aocc-4.0/lib_ILP64/libblis-mt.a"  ]] &&
```

Comment:
For 4.0 did the include tree also change does it match BLIS_INCLUDE_DIR?   Not sure if you tested with that install.  I am not sure if you even need to support older than 4.2 as you are adding new dependency.   Usually can just what is advises in the documentation.

Reply:
If BLIS_LIB is `/opt/AMD/aocl/aocl-linux-aocc-4.0/lib_ILP64/libblis-mt.a` 
BLIS_INCLUDE_DIR will be `/opt/AMD/aocl/aocl-linux-aocc-4.0/include_ILP64/`
which seems to be right, i haven't tested it though. I am okay with changing it to just use 4.2.